egs/aspire/s5/local/multi_condition/aspire_prep_rir_noise_list.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to generate rir_list and noise_list in aspire
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import argparse, glob, math, os, sys
+
+
+def GetArgs():
+    parser = argparse.ArgumentParser(description="Prepare rir_list and noise_list for Aspire  "
+                                                 "Usage: reverberate_data_dir.py [options...] <in-data-dir> <out-data-dir> "
+                                                 "E.g. reverberate_data_dir.py "
+                                                 "data/impulses_noises data/impulses_noises/info",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("input_dir", help="Input data directory")
+    parser.add_argument("output_dir", help="Output data directory")
+    print(' '.join(sys.argv))
+    args = parser.parse_args()
+
+    return args
+
+
+# This function generate the rir_list file for the aspire real RIR
```

Comment:
`This function generates the rir_list file for the real RIRs being in ASpIRE experiments. It assumes the availability of data/impulses_noises directory prepared by local/multi_condition/prepare_impulses_noises.sh.`